README.md

```diff
@@ -73,7 +73,7 @@ The generation process uses the predefined `CommonModel`s from the input stage t
 
 Check out [the example](#example) to see how to use the library and [generators document](./docs/generators.md) for more info.
 
-> **NOTE**: Each implemented language has different options, dictated by the nature of the language. Keep this in mind when selecting a language.
+> **NOTE**: Each implemented language has different options, dictated by the nature of the language, so the output can vary in support between languages.
```

Comment:
```suggestion
> **NOTE**: Each implemented language has different options, dictated by the nature of the language, so the output can vary between languages.
```

Reply:
Maybe we should remove the last part `, so the output can vary between languages.`. As it is kinda obvious that it varies since languages vary 😆

Reply:
We can :)